### PR TITLE
Expand testing to include `to_pysat()` and Improved Standards Compliance

### PR DESCRIPTION
**Pull request by rstoneback**
I improved the overall package standards, and brought build and other processes up to current times. Passes tests with flake8.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,62 @@
+[metadata]
+name = pysatCDF
+version = 0.3.1
```
Comment (jklenzing) [inline setup.cfg:3]: can point to version file with 
```suggestion
version = file: pysatCDF/version.txt
```

**Comment by rstoneback** [comment]
I ran into a metadata issue trying to run the pysatModels match example but it works now.

**Comment by JonathonMSmith** [comment]
This works for me. My only concern with a move back to pysatCDF in its current form is that it contains the CDF code. And it's using the UNIX compatible version. This means getting this running on windows is a huge pain which is a chronic hurdle for hackathons. This is out of the scope of this pull, but finding a way to incorporate or work around the CDF installations at:
https://spdf.gsfc.nasa.gov/pub/software/cdf/dist/cdf37_1/windows/ would be useful for a smoother setup for all types of users. I think this requires some discussion before I write up an issue though

**Comment by rstoneback** [comment]
I'm all for getting pysatCDF up to the latest CDF library and, presuming I can get the multi-dimensional support going, ensuring it runs via windows and WSL. This isn't the beginning of the end, it is the end of the beginning. In the meantime, I'll need an all clear here before I can improve anything else 😈

**Pull request merged**